docs/source/format/Columnar.rst

```diff
@@ -566,11 +572,6 @@ having the values: ``[{f=1.2}, null, {f=3.4}, {i=5}]``
 ::
 
     * Length: 4, Null count: 1
-    * Validity bitmap buffer:
-      |Byte 0 (validity bitmap) | Bytes 1-63            |
-      |-------------------------|-----------------------|
-      |00001101                 | 0 (padding)           |
-
     * Types buffer:
 
       |Byte 0   | Byte 1      | Byte 2   | Byte 3   | Bytes 4-63  |
```

Comment:
It seems the types and offsets buffers need to be fixed (the second entry isn't unspecified anymore).

Reply:
Oops, went to fast. Fixing